### DISCUSSION TITLE
Matrix Input for Get MAtrix Slice Slice Selector

### DISCUSSION THREAD

**Discussion by jab0707** [SCIInstitute/SCIRun#2228] [Q&A]
It would be nice if I could enter a matrix containing a single number to select the slice index for a get matrix slice. For instance if I have a 10 x 15 matrix, and I want to grab column 5, I would input a second matrix with a single entry with a value of 5.

**Comment by jessdtate** [comment]
The scalar input does this (light blue port). ConvertMatrixToScalar will allow you to use a matrix type.

**Comment by dcwhite** [comment]
@jab0707 I propose to turn this one into a discussion rather than an issue.
